Project_Mira_DaU/Source/Project_Mira_DaU/Public/Game/GameManager.h

```diff
@@ -0,0 +1,50 @@
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "Blueprint/UserWidget.h"
+#include "GameManager.generated.h"
+
+UCLASS()
+class PROJECT_MIRA_DAU_API AGameManager : public AActor
```

Comment:
ça m'a l'air d'être une façon "Unity" de faire les choses.

https://dev.epicgames.com/documentation/en-us/unreal-engine/gameplay-framework-in-unreal-engine
https://dev.epicgames.com/documentation/en-us/unreal-engine/game-mode-and-game-state-in-unreal-engine

ça ne te semble pas être du code qui parait pertinent dans une classe héritant de AGameMode ? (ou sinon AGameState pour gérer le timer et APlayerState pour gérer le nombre de ressources collectées par le joueur)

"The game mode is instantiated immediately after your level is loaded in the engine and the world is created. The game mode is a server-based manager class inherited from the actor class. Since this class is created upon level load, it is not persistent across levels. Game mode is the first actor to instantiate upon level load and can be set on map-by-map basis. Game mode exists at the heart of the gameplay framework managing the overall rules and structure of a gameplay session and instantiating the remaining framework actors upon creation. The first two are game state and player state.
...
Game mode is the primary class that specifies which other classes to use in the gameplay framework and is commonly used to specify game rules for modes, such as capture the flag."